### PR TITLE
Added instrumentation -> Benchmarks submodule for JMH benchmarks

### PR DESCRIPTION
**Pull request by timurgen**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
-target/
-dependency-reduced-pom.xml
+target/
+dependency-reduced-pom.xml
+/config/nbproject/
```
Comment (jeqo) [inline .gitignore]: nit: could you add this on your global git ignore? https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

**Review by jeqo — CHANGES_REQUESTED** [review]
Looks great! I will test it further as soon as I have time. Small comments.

File: .gitignore (hunk 1 above)
Comment (timurgen) [reply .gitignore]: ok

**Review by jeqo — COMMENTED** [review]
Could the `benchmark` module be moved one level up? Instrumentation module is not necessary IMO.

**Comment by timurgen** [comment]
@jeqo fixed

File: .gitignore (hunk 1 above)
Comment (jeqo) [reply .gitignore]: @timurgen by adding `/config/nbproject/` to your global gitignore, then it could be removed from here :)

**Review by jeqo — APPROVED** [review]

**Comment by jeqo** [comment]
@timurgen I made small fixes: remove IT and update .gitignore according to comments. I will merge this now.

**Pull request merged**